website/www/site/content/en/contribute/release-guide.md

```diff
@@ -230,7 +231,19 @@ If you are not a PMC, please ask for help in dev@ mailing list.
 **********
 
 
-## 3. Investigate performance regressions
+## 3. Update base image dependencies for Python container images
+
+1. Check the versions specified in sdks/python/container/base_image_requirements_manual.txt` and update them if necessary.
+2. Regenerate full dependency list by running:
```

Comment:
prefer 1. for numbered lists in markdown.

Reply:
done, thanks